src/libs/tmdk/e2e-tests/basic-e2e/basic-e2e.test.ts

```diff
@@ -0,0 +1,492 @@
+      workspaceId: constants.workspaceId,
+      componentTypeId: constants.componentType1Input.componentTypeId,
+    });
+  expect(componentType1Result.componentTypeId).toEqual(
+    constants.componentType1Input.componentTypeId
+  );
+  expect(componentType1Result.status).toMatchObject({
+    state: constants.resourceActiveState,
+  });
+  const componentType2Result: GetComponentTypeCommandOutput =
+    await aws().tm.getComponentType({
+      workspaceId: constants.workspaceId,
+      componentTypeId: constants.componentType2.componentTypeId,
+    });
+  expect(componentType2Result.componentTypeId).toEqual(
+    constants.componentType2.componentTypeId
+  );
+  expect(componentType2Result.status).toMatchObject({
+    state: constants.resourceActiveState,
+  });
+  // Verify Entities
+  const entity1Result = await aws().tm.getEntity({
+    workspaceId: constants.workspaceId,
+    entityId: entity1Id,
+  });
+  expect(entity1Result.entityId).toEqual(entity1Id);
+  expect(entity1Result.entityName).toEqual(constants.entity1Input.entityName);
+  expect(entity1Result.status!.state).toBe(constants.resourceActiveState);
+  const listEntitiesResult: EntitySummary[] = (
+    await aws().tm.listEntities({ workspaceId: constants.workspaceId })
+  ).entitySummaries!;
+  listEntitiesResult.forEach(function (entity) {
+    if (entity.entityName === constants.entity2Definition.entityName) {
+      expect(entity.status!.state).toBe(constants.resourceActiveState);
+    }
+  });
+  // Verify Scenes
+  const scene1Result = await aws().tm.getScene({
+    workspaceId: constants.workspaceId,
+    sceneId: constants.scene1Input.sceneId,
+  });
+  expect(scene1Result.contentLocation).toBe(scene1S3Location);
+  const scene1S3Response = await aws().s3.getObject({
+    Bucket: workspaceS3BucketName,
+    Key: constants.scene1FileName,
+  });
+  const scene1JsonResult = JSON.parse(
+    await scene1S3Response.Body!.transformToString(constants.jsonEncoding)
+  );
+  expect(scene1JsonResult.nodes[0].name).toBe(
+    constants.model1FileName.replace(".glb", "")
+  );
+  const scene2Result = await aws().tm.getScene({
+    workspaceId: constants.workspaceId,
+    sceneId: constants.scene2FileName.replace(".json", ""),
+  });
+  expect(scene2Result.contentLocation).toBe(
+    scene1S3Location.replace(constants.scene1FileName, constants.scene2FileName)
+  );
+  const scene2S3Response = await aws().s3.getObject({
+    Bucket: workspaceS3BucketName,
+    Key: constants.scene2FileName,
+  });
+  const scene2JsonResult = JSON.parse(
+    await scene2S3Response.Body!.transformToString(constants.jsonEncoding)
+  );
+  const scene2ModelNames: string[] = [];
+  scene2JsonResult.nodes.forEach(function (model: { name: string }) {
+    scene2ModelNames.push(model.name);
+  });
+  expect(scene2ModelNames).toContain(
+    constants.model1FileName.replace(".glb", "")
+  );
+  expect(scene2ModelNames).toContain(
+    constants.model2FileName.replace(".glb", "")
+  );
+  // Verify Models
```

Comment:
should we instead compare against the `basic-e2e-resources` ?

Reply:
This is specifically checking whether the scene definition contains the correct model name. I could have used a const to store the model name but it was easier to just strip the file extension from the already saved file name